akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterShardingSettings.scala

```diff
@@ -114,12 +125,53 @@ object ClusterShardingSettings {
       val updatingStateTimeout: FiniteDuration,
       val entityRecoveryStrategy: String,
       val entityRecoveryConstantRateStrategyFrequency: FiniteDuration,
-      val entityRecoveryConstantRateStrategyNumberOfEntities: Int) {
+      val entityRecoveryConstantRateStrategyNumberOfEntities: Int,
+      val coordinatorStateWriteMajorityPlus: Int,
+      val coordinatorStateReadMajorityPlus: Int) {
 
     require(
       entityRecoveryStrategy == "all" || entityRecoveryStrategy == "constant",
       s"Unknown 'entity-recovery-strategy' [$entityRecoveryStrategy], valid values are 'all' or 'constant'")
 
+    def this(
```

Comment:
`// included for backwards compatibility` + `@deprecated` ?

Reply:
good call, made me find that it should also be added in sharding-typed